client/src/views/v2/DownloadFilesView.js

```diff
@@ -26,7 +26,7 @@ export default function DownloadFilesView() {
 
   return (
     <ContentPage title="Files">
-      {files.length > 0 && (
+      {files !== null && files.length > 0 && (
```

Comment:
Could use null chaining here for brevity. `files?.length > 0`

Reply:
Great idea, love this syntax!